### PR TITLE
Fix sn reponse time

### PR DESCRIPTION
**Pull request by l-vincent-l**
When launch a dijkstra we were always tring to relaunch a dijkstra on stop point that we didn't touch, so that was taking a lot of time.

### CODE REVIEW

**Comment by buildbotctp** [comment]
Can one of the admins verify this patch?

**Comment by l-vincent-l** [comment]
Le ven. 14 mars 2014 14:58:11 CET, buildbotctp a écrit :

> Can one of the admins verify this patch?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CanalTP/kraken/pull/199#issuecomment-37649260.

nop

File: source/kraken/worker.cpp [hunk 1]
```diff
@@ -387,7 +387,7 @@ void Worker::init_worker_data(){
 }
 
 
-pbnavitia::Response Worker::pt_ref(const pbnavitia::PTRefRequest &request) {
+pbnavitia::Response Worker::pt_ref(const pbnavitia::PTRefRequest &request){
```
Comment (antoine-de) [inline source/kraken/worker.cpp:390]: rahhh pourquoi avoir enlevé l'espace ?

**Comment by antoine-de** [comment]
test this please

**Comment by l-vincent-l** [comment]
Le ven. 14 mars 2014 16:14:14 CET, Antoine D a écrit :

> Merged #199 https://github.com/CanalTP/kraken/pull/199.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/CanalTP/kraken/pull/199.

Ce n'était pas dans guideline l'espace.

**Pull request merged**